### ISSUE TITLE
Source(s) files not rsynced to production

### ISSUE THREAD

**Issue by gjost** [denshoproject/encyc-core#25]
A Source object in the production encyclopedia has no image: http://encyclopedia.densho.org/sources/en-igamori-4/

The image is present and visible in the PSMS: https://psms.densho.org/admin/sources/source/2512/change/?_changelist_filters=q%3Den-igamori-4

`encyc sources` syncs *metadata* to Elasticsearch and *binaries* to the production image server.  In this case the metadata is being synced but the binary is not.

```
(encyccore) encyc@fuzzy:/opt/encyc-core$ encyc sources --sourceid=en-igamori-4
Debug mode is off
2020-01-21 12:51:32.216784 hosts: [{'host': 'REDACTED', 'port': '9200'}]
2020-01-21 12:51:32.217307 index: encyc-20180226c
2020-01-21 12:51:32.217400 ------------------------------------------------------------------------
2020-01-21 12:51:32.217439 getting sources from PSMS...
2020-01-21 12:51:42.222271 psms sources: 1205
2020-01-21 12:51:42.222325 getting sources from Elasticsearch...
2020-01-21 12:51:43.250431 es_sources: 1204
2020-01-21 12:51:43.251164 adding sources...
2020-01-21 12:51:43.251209 --------------------
2020-01-21 12:51:43.251226 1/1 en-igamori-4
2020-01-21 12:51:43.251242 <legacy.Source 'en-igamori-4'>
2020-01-21 12:51:43.251266 getting from Elasticsearch
WARNING:elasticsearch:GET http://REDACTED:9200/encyc-20180226c/sources/%3Clegacy.Source+%27en-igamori-4%27%3E [status:404 request:0.002s]
2020-01-21 12:51:43.253210 not in ES
2020-01-21 12:51:43.253607 <elastic.Source 'en-igamori-4'>
2020-01-21 12:51:43.253662 saving
2020-01-21 12:51:43.275815 original_path_abs /var/www/encycpsms/media/sources/sources/N/None/en-igamori-4_Qm5UXFg.jpg
2020-01-21 12:51:43.275880 display_path_abs /var/www/encycpsms/media/sources/sources/N/None/en-igamori-4_KeavnCu.jpg
2020-01-21 12:51:43.275909 ok
2020-01-21 12:51:43.275922 --------------------
2020-01-21 12:51:43.275932 rsyncing
2020-01-21 12:51:43.275942 /var/www/encycpsms/media/sources... -> encyc@REDACTED:/var/www/media/encyc-psms/
2020-01-21 12:51:43.646282 - /var/www/encycpsms/media/sources/sources/N/None/en-igamori-4_KeavnCu.jpg (missing)
2020-01-21 12:51:43.646421 - /var/www/encycpsms/media/sources/sources/N/None/en-igamori-4_Qm5UXFg.jpg (missing)
2020-01-21 12:51:43.646467 No local_paths: []
2020-01-21 12:51:43.646487 ========================================================================
2020-01-21 12:51:43.646515 Files missing from local fs: [u'/var/www/encycpsms/media/sources/sources/N/None/en-igamori-4_KeavnCu.jpg', u'/var/www/encycpsms/media/sources/sources/N/None/en-igamori-4_Qm5UXFg.jpg']
2020-01-21 12:51:43.646573 ERROR: 1 titles were unpublishable:
2020-01-21 12:51:43.646596 ERROR: Could not upload No local_paths: []
2020-01-21 12:51:43.646633 DONE
2020-01-21 12:51:43.654826 TIME: 0:00:11.438054
```

**Comment by gjost** [comment]
I manually synced the photos to the image server.  Obviously this does not solve the problem of why they weren't syncing automatically. (from PSMS dir on editors mediawiki server to production media server)

**Comment by gjost** [comment]
Fixed in `encyc-core` commit `9648b2c` for package `encyccore-master_3.0.1`.
